src/battle_game/core/bullets/rocket.cpp

```diff
@@ -14,29 +16,88 @@ Rocket::Rocket(GameCore *core,
                glm::vec2 velocity)
     : Bullet(core, id, unit_id, player_id, position, rotation, damage_scale),
       velocity_(velocity) {
+  auto &units = game_core_->GetUnits();
+  auto player = game_core_->GetPlayer(player_id_);
+  auto &input_data = player->GetInputData();
+  auto WhereCursorIs = input_data.mouse_cursor_position;
```

Comment:
variable names should be in lower_case format

Reply:
Thanks. Fixed